tests/StringHelperTest.php

```diff
@@ -401,4 +401,11 @@ public function testUcwords(string $string, string $expectedResult): void
     {
         $this->assertSame($expectedResult, StringHelper::ucwords($string));
     }
+
+    public function testTruncateBegin(): void
+    {
+        $this->assertSame('sms sent to …456.', 'sms sent to ' . StringHelper::truncateBegin('123456', 3) . '.');
```

Comment:
I think it's 3 identical test cases. Is it right?
Remove the last dot from the strings and remove 2 last cases

Reply:
@xepozz I removed the last 2 tests and add one more for test $suffix variable ¿it's ok?.